spec/models/spree/shipment_spec.rb

```diff
@@ -57,7 +57,7 @@
     end
 
     context "variant was removed" do
-      before { variant.product.destroy }
+      before { variant.reload.product.destroy }
```

Comment:
I still wonder why this is necessary. Is the variant or the product out of date?

Reply:
It looks like it isn't needed. I'll drop it :+1: